frontend/javascripts/oxalis/view/context_menu.js

```diff
@@ -295,18 +309,20 @@ function NodeContextMenuOptions({
       >
         Delete this Node {activeNodeId === clickedNodeId ? shortcutBuilder(["Del"]) : null}
       </Menu.Item>
-      <Menu.Item
-        className="node-context-menu-item"
-        key="measure-node-path-length"
-        disabled={activeNodeId == null || !areInSameTree || isTheSameNode}
-        onClick={() =>
-          activeNodeId != null
-            ? measureAndShowLengthBetweenNodes(activeNodeId, clickedNodeId)
-            : null
-        }
-      >
-        Path Length to this Node
-      </Menu.Item>
+      {isTheSameNode ? null : (
```

Comment:
It bugged me that rightclicking a selected node always showed "Path length to this node: 0 vx".